### ISSUE TITLE
Breaking stacked berry bushes causes server exception and player kick

### ISSUE THREAD

**Issue by itsTheFae** [anegostudios/VintageStory-Issues#2191]
**Game Version:** 1.17.3  (world generated in 1.17.2)
**Platform:** Windows & Linux   
**Modded:** No   
**SP/MP:** Multiplayer (linux server, windows and linux clients)  

### Description
Berry bushes cause an exception when cut down from the bottom, resulting in player kick.
This bug has been reported by at least one other player on the discord:  https://discord.com/channels/302152934249070593/334233095374897152/1017842710624874518

### How to reproduce
1. Plant berry bushes, stacking them.
2. Allow berries to ripen.
3. Attempt to cut berries down using bottom-most bush and an axe (perhaps other tools)
4. Experience client kick due to unexpected exception.

### Expected behavior
When tall, ripe bushes are cut down from the bottom, they should drop the top and bottom bushes and berries as items without kicking the player. 

### Logs
Exception from `server-main.txt`:
```
13.9.2022 12:01:52 [Event] Player [Redacted] got removed. Reason: Threw an exception at the server
13.9.2022 12:01:52 [Error] System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.GameContent.BlockBerryBush.GetDrops (Vintagestory.API.Common.IWorldAccessor world, Vintagestory.API.MathTools.BlockPos pos, Vintagestory.API.Common.IPlayer byPlayer, System.Single dropQuantityMultiplier) [0x00043] in <f01e2cfb17af4d6e8438cdb4242c5d13>:0 
  at Vintagestory.API.Common.Block.OnBlockBroken (Vintagestory.API.Common.IWorldAccessor world, Vintagestory.API.MathTools.BlockPos pos, Vintagestory.API.Common.IPlayer byPlayer, System.Single dropQuantityMultiplier) [0x00055] in <3cdbf5280d60457f85295e780f5884f1>:0 
  at Vintagestory.Common.BlockAccessorBase.BreakBlock (Vintagestory.API.MathTools.BlockPos pos, Vintagestory.API.Common.IPlayer byPlayer, System.Single dropQuantityMultiplier) [0x00007] in <6215ae1029cd4696b4c4b2c34106b061>:0 
  at Vintagestory.GameContent.BlockPlant.OnNeighbourBlockChange (Vintagestory.API.Common.IWorldAccessor world, Vintagestory.API.MathTools.BlockPos pos, Vintagestory.API.MathTools.BlockPos neibpos) [0x00015] in <f01e2cfb17af4d6e8438cdb4242c5d13>:0 
  at Vintagestory.Server.ServerMain.TriggerNeighbourBlocksUpdate (Vintagestory.API.MathTools.BlockPos pos) [0x0006a] in <6215ae1029cd4696b4c4b2c34106b061>:0 
  at Vintagestory.Server.ServerSystemBlockSimulation.HandleBlockPlaceOrBreak (Packet_Client packet, Vintagestory.Server.ConnectedClient client) [0x001ca] in <6215ae1029cd4696b4c4b2c34106b061>:0 
  at Vintagestory.Server.ServerMain.HandleClientPacket (Vintagestory.Server.ConnectedClient client, System.Byte[] data) [0x0005a] in <6215ae1029cd4696b4c4b2c34106b061>:0 
  at Vintagestory.Server.ServerMain.ProcessNetMessage (Vintagestory.Common.NetIncomingMessage msg, Vintagestory.Common.NetServer mainSocket) [0x00151] in <6215ae1029cd4696b4c4b2c34106b061>:0 

```

**Comment by wasakoki** [comment]
this also happens on both short and tall bushes if you break the soil block that its planted on

**Comment by tyronx** [comment]
thanks for the report, i was able to reproduce this now. will be fixed in 1.17.4
